src/components/Navbar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+  <div className="nav-bar">
+    <nav className="nav container-width">
+      <div>
+        <h1>Math Magicians</h1>
+      </div>
+      <div className="links">
+        <Link to="/">Home</Link>
+        <Link to="/calculator">Calculator</Link>
+        <Link to="/quote">Quotes</Link>
```

Comment:
- [ ] [OPTIONAL] You did a great job with routing and navigation. :+1: React router provides a special type of `Link` called `NavLink` which knows whether it is active or not. It makes it easier to apply styles on the currently active link. 😎️ Check the React router [documentation](https://reactrouter.com/en/main/components/nav-link) for more information.